File: Volume 113/11364 - Parking/11364 - Parking.java
import java.util.Scanner;

public class Parking{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int rep = in.nextInt();
        for(int i = 0; i < rep; i++){
            int numOfStores = in.nextInt();
            int [] coords = new int [numOfStores];
            int ttl = 0;
            for(int j = 0; j < numOfStores; j++){
                coords[j] = in.nextInt();
                ttl += coords[j];
            }
            int med = ttl/numOfStores, dist = 0;
            for(int j = 0; j < numOfStores; j++){
                dist += Math.max(coords[j], med) - Math.min(coords[j], med);
            }
            System.out.println(dist);
        }
    }
}

File: Volume 113/11364 - Parking/11364 - Parking.cpp
#include <iostream>
#include <algorithm>
using namespace std;

int main(){
    int rep;
    cin >> rep;
    for(int i = 0; i < rep; i++){
        int numOfStores; cin >> numOfStores;
        int coords[numOfStores];
        for(int j = 0; j < numOfStores; j++) cin >> coords[j];
        sort(coords, coords + numOfStores);
        cout << 2*(coords[numOfStores - 1] - coords[0]) << endl;
    }
}